.github/workflows/CFF-test.yml

```diff
@@ -0,0 +1,21 @@
+
+name: Checking CITATION.cff
+
+on:
+  push:
+    paths:
+      - "CITATION.cff"
+  pull_request:
+    paths:
+      - "CITATION.cff"
+
+jobs:
+  cffconvert:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          persist-credentials: false
+      - uses: citation-file-format/cffconvert-github-action@2.0.0
```

Comment:
Interesting... Can we be sure that this Action will be maintained? Which company/institution manages this?

Reply:
The Action is Docker image of the corresponding Python 3 CLI `cffconvert`.  The team behind the CLI also maintains the Action and researches on release automation, at the moment, such that each new release of the CLI would also update and release the Action automatically.  The CLI is expected to receive updates when the CFF version is updated.  Until then, there should be no changes to both the CLI and the Action.

Reply:
Thank you for the clarifications! 😸